app/assets/stylesheets/darkswarm/menu.css.scss

```diff
@@ -36,12 +36,12 @@ nav {
   .top-bar-section .has-dropdown > a {
     padding-right: ($topbar-height / 3) !important;
 
-    i.ofn-i_022-cog, i.fa{
+    i.ofn-i_022-cog, .ofn-i_071-globe {
```

Comment:
Great! We should add to the testing notes that we need to check that all symbols still align correctly.